.gitignore

```diff
@@ -62,6 +62,7 @@ dkms.conf
 # autotools
 /aclocal.m4
 /autom4te.cache/
+/build/
```

Comment:
Since https://github.com/mity/md4c/wiki/Building-MD4C suggests using this directory